source/2020-02-28-the-ember-times-issue-137.md

```diff
@@ -81,17 +81,23 @@ Submissions will be accepted **until March 8th**, but there’s no need to wait!
 
 We look forward to your participation! If you have any immediate questions, feel free to email the survey team via survey@emberjs.com, or ping them in #dev-ember-learning on the [Discord chat](https://discordapp.com/invite/emberjs).
 
-But please, don't forget to help us spread the word by sharing the survey landing page on your social network feeds, at meetups and around your office and other communities. 
+But please, don't forget to help us spread the word by sharing the survey landing page on your social network feeds, at meetups and around your office and other communities.
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
+## [Can't wait until Ember 3.17 is out? Become a beta tester today 🧪](https://github.com/ember-cli/ember-cli/releases/tag/v3.17.0-beta.1)
+
+It's not too long until [Ember 3.17 is out](https://emberjs.com/releases/) and until you can upgrade your favourite
+Ember app to the latest version of the framework. If you want to have an extra head start, you can even **try out** some of
+**the 3.17 Ember experience** today!
+
+A few days ago, [ember-cli@3.17.0-beta.1](https://github.com/ember-cli/ember-cli/releases/tag/v3.17.0-beta.1) has been published with lots of awesome goodies, including the removal of [RSVP in favour of native promises](https://github.com/ember-cli/ember-cli/pull/9041), lots of great updates to test scenarios and a linter level up with [the now direct usage of eslint and ember-template-lint](https://github.com/ember-cli/ember-cli/pull/9009). Your feedback from testing the ember-cli beta is greatly appreciated. Share your thoughts either on [the project repository on Github](https://github.com/ember-cli/ember-cli/) or in the [#ember-cli channel on the Ember Discord](https://discordapp.com/invite/emberjs).
```

Comment:
As you added "a few days ago," I thought the past tense "was published" would sound better than "has been published".
```suggestion
A few days ago, [ember-cli@3.17.0-beta.1](https://github.com/ember-cli/ember-cli/releases/tag/v3.17.0-beta.1) was published with lots of awesome goodies, including the removal of [RSVP in favour of native promises](https://github.com/ember-cli/ember-cli/pull/9041), lots of great updates to test scenarios and a linter level up with [the now direct usage of eslint and ember-template-lint](https://github.com/ember-cli/ember-cli/pull/9009). Your feedback from testing the ember-cli beta is greatly appreciated. Share your thoughts either on [the project repository on Github](https://github.com/ember-cli/ember-cli/) or in the [#ember-cli channel on the Ember Discord](https://discordapp.com/invite/emberjs).
```